src/hooks/useAggregatorStats.ts

```diff
@@ -0,0 +1,58 @@
+import useSWR from 'swr'
+import { ChainId } from '@kyberswap/ks-sdk-core'
+
+type Response = {
+  pools: Record<
+    string,
+    {
+      poolSize: number
+      tvl: number
+      tokenSize: number
+    }
+  >
+}
+
+const chainIdMapping: Partial<Record<ChainId, string>> = {
+  [ChainId.MAINNET]: 'ethereum',
+  [ChainId.MATIC]: 'polygon',
+  [ChainId.BSCMAINNET]: 'bsc',
+  [ChainId.AVAXMAINNET]: 'avalanche',
+  [ChainId.FANTOM]: 'fantom',
+  [ChainId.CRONOS]: 'cronos',
+  [ChainId.ARBITRUM]: 'arbitrum',
+  [ChainId.BTTC]: 'bttc',
+  [ChainId.VELAS]: 'velas',
+  [ChainId.AURORA]: 'aurora',
+  [ChainId.OASIS]: 'oasis',
+  [ChainId.OPTIMISM]: 'optimism',
```

Comment:
Why dont we use directly from `NETWORK_INFOS.route`

Reply:
Just because they're different in just BSCMAINNET, I need 'bsc', while we had 'bnb'

Reply:
okay, just saw Danh's cmt